perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -112,7 +112,10 @@ sub send() {
                     ( $last_problem_state eq '' && $row->{item_problem_state} eq 'confirmed' ) ||
                     $last_problem_state ne $row->{item_problem_state}
                 ) ) {
-                    my $update = sprintf(_('Problem marked as %s'), $row->{item_problem_state});
+                    my $update = sprintf(
+                        _('Problem marked as %s'),
```

Comment:
For translation ease, probably best if we go with _('State changed to:') as the template.

Reply:
👍